README.md

```diff
@@ -1,18 +1,16 @@
-docker-ansible : using ansible 2 without installing it
+docker-ansible : using ansible without installing it
 ======================================================
 
-This docker allows you to use ansible 2 on your computer without installing it.
+This docker allows you to use ansible on your computer without installing it.
 
 This allows you to :
 
 * share your ssh agent with the ansible docker
-* disables known_hosts checking
-* share /etc/hosts with ansible
 
 Just add these aliases to your ~/.bash_aliases in order to use ansible as it where installed on your computer.
 
 ```bash
-alias base_ansible='docker run -it --rm --volume $SSH_AUTH_SOCK:/ssh-agent --env SSH_AUTH_SOCK=/ssh-agent -v /etc/hosts:/etc/hosts:ro -v ${PWD}:${PWD} -v ${HOME}/.ssh:/root/.ssh --env ANSIBLE_HOST_KEY_CHECKING=False -w ${PWD} kitpages/docker-ansible'
+alias base_ansible='docker run -it --rm --volume $SSH_AUTH_SOCK:/ssh-agent --env SSH_AUTH_SOCK=/ssh-agent -v ${PWD}:${PWD}     -v ${HOME}/.ssh/known_hosts:/root/.ssh/known_hosts -w ${PWD} kitpages/docker-ansible'
```

Comment:
In this doc, you shoud add the version of ansible at the end of the alias (in order to be clearer for the user).

Perhaps an even better way would be to add a variable in the bash_aliases

```bash
export DOCKER_ANSIBLE_VERSION=2.2
alias base_ansible='[......] kitpages/docker-ansible:${DOCKER_ANSIBLE_VERSION}'
```
or something like that.